miio/vacuum.py

```diff
@@ -540,6 +540,21 @@ def get_room_mapping(self):
         """Retrieves a list of segments."""
         return self.send("get_room_mapping")
 
+    @command()
+    def get_recover_maps(self):
+        """Get recover maps."""
+        return self.send("get_recover_maps")
+
+    @command(
+        click.argument("id"),
+        click.argument("map"),
+    )
+    def recover_map(self, id: int, map: int):
+        """Set recover maps."""
```

Comment:
`use_saved_map` in accordance to the above command? What are ID and map in this case? Can you post an example of what get recover maps returns?

Reply:
Example:
```
miiocli vacuum --ip roboip --token robotoken get_recover_maps
Running command get_recover_maps
```
returns:
```
[[1, 1575723134], [2, 1577540948], [3, 1577807926]]
```
I named the first number (1,2,3) as id and the second (1575723134,1577540948, 1577807926) as map

So in my case the command:
```
miiocli vacuum --ip roboip --token robotoken recover_map 1 1575723134
Running command recover_map
Setting the map [1, 1575723134] as active
ok
```
sets the initial (not edited) map as active:
![PNG image](https://user-images.githubusercontent.com/15311791/71663805-e889be00-2d56-11ea-9a74-d7d9f4929999.png)

```
miiocli vacuum --ip roboip --token robotoken recover_map 2 1577540948
Running command recover_map
Setting the map [2, 1577540948] as active
ok
```
sets the oldest backup map (the one from 2019.12.28 14:49, at the very bottom of the screeny, map not fully visible) as active:
![PNG image 1](https://user-images.githubusercontent.com/15311791/71664043-c17fbc00-2d57-11ea-9909-7589c2b8169a.png)

```
miiocli vacuum --ip roboip --token robotoken recover_map 3 1577807926
Running command recover_map
Setting the map [3, 1577807926] as active
ok
```
sets the other backup map (the one from 2019.12.31 16:58) as active:
![PNG image 2](https://user-images.githubusercontent.com/15311791/71664306-d0b33980-2d58-11ea-9b6d-598f0a336bd1.png)


So I suggest to name it `use_recover_map` according to `get_recover_maps`?